components/ProductsList/index.js

```diff
@@ -1,12 +1,14 @@
-import { uid } from "uid";
 import ProductListItem from "../ProductListItem";
 export default function ProductsList({ products }) {
   return (
     <>
       <h2>Product List</h2>
       <ul>
         {products.map((product) => (
-          <ProductListItem key={uid()} product={product}></ProductListItem>
+          <ProductListItem
+            key={product._id}
+            product={product}
+          ></ProductListItem>
```

Comment:
If a component doesn't have any children (content between the opening and closing tags), you can use a self-closing tag for simplicity and readability. 

```suggestion
          <ProductListItem
            key={product._id}
            product={product}
          />
```

Reply:
Good point. We changed it.